scripts/optimize_images/README.md

```diff
@@ -0,0 +1,36 @@
+# Optimizing Static Images
+
+Static images can take a long time to load on web pages. In order to make this
+faster, we can optimize them in two ways:
+
+- Reduce the image resolution to be appropiate to most devices
```

Comment:
Can you explain the difference between reducing resolution and reducing dimensions?

Also, appropiate => appropriate 😛

Reply:
Updated - we are just reducing the dimensions in the script, CRA doesn't do anything else actually.